Controller/Adminhtml/Importcategory/Save.php

```diff
@@ -71,7 +71,7 @@ public function __construct(
     }
 
     /**
-     * run the action
+     * Imports categories using csv file
      *
      * @return \Magento\Backend\Model\View\Result\Redirect
```

Comment:
Use `use` statement to keep PHPDoc blocks concise and readable.

Reply:
The`use` statement belongs at the top of the file with the existing one on line 29. Then you can change the return in your PHPDoc to:
`@return Redirect`